experiment/secret-sync-controller/BUILD.bazel

```diff
@@ -0,0 +1,43 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "main.go",
+        "secret_sync_spec.go",
```

Comment:
```suggestion
        "secret-sync-config.go",
```

Reply:
Oh.. sorry for that, I was being stupid.
Forgot to rerun it before I push